src/python/pants/backend/python/rules/pex_test.py

```diff
@@ -298,6 +303,26 @@ def test_additional_args(self) -> None:
         pex_info = self.create_pex_and_get_pex_info(additional_pex_args=("--not-zip-safe",))
         assert pex_info["zip_safe"] is False
 
+    def test_platforms(self) -> None:
```

Comment:
Can this test run on MacOS? If not we should add the appropriate filtering annotation.

Reply:
Yes, it can. I developed it on macOS and chose `cryptography` very carefully because it solely uses prebuilt wheels/bdists. 

The only danger with `platforms` is when you encounter sdists.